source/helpers/selector-observer.tsx

```diff
@@ -55,5 +57,5 @@ export default function observe<
 	signal?.addEventListener('abort', () => {
 		rule.remove();
 	});
-	window.addEventListener('animationstart', getListener(seenMark, selector, listener), {signal});
```

Comment:
This lets us listen to the feature unload inside the observer’s callback:

```jsx
function add(element, {signal}) {
	wrap(element, <strong/>)
	signal.addEventListener('abort', () => {
		element.remove();
	});
}

function init(signal) {
	observe('a', add, {signal})
}
```

Mentioned in:

- #6554